granian/cli.py

```diff
@@ -195,6 +195,11 @@ def option(*param_decls: str, cls: Optional[Type[click.Option]] = None, **attrs:
     type=click.IntRange(60),
     help='The maximum amount of time in seconds a worker will be kept alive before respawn',
 )
+@option(
+    '--workers-graceful-timeout',
+    default=30.0,
```

Comment:
I would make this optional, to keep the existing behaviour as default.
Also, using an integer instead of a float and `click.IntRage` as type to limit the accepted values range seems better to me.